week-1/K-functions-parameters/exercise.js

```diff
@@ -1,5 +1,6 @@
 // Complete the function so that it takes input parameters
 function multiply() {
+  return 3 * 4;
```

Comment:
This `multiply` function is supposed to take two input parameters. It should be something like:
```javascript
function multiply(num1, num2) {
  return num1 * num2;
}
```
This way, you can do `multiply(3, 4)` and `multiply(6, 8)` and the function will always use the given numbers.